src/server/qtquick/wsurfaceitem.cpp

```diff
@@ -1307,6 +1296,11 @@ qreal WSurfaceItemPrivate::getImplicitHeight() const
     return surfaceState->contentGeometry.height() + ps.height();
 }
 
+bool WSurfaceItem::isActive() const
```

Comment:
Rename to `isValid()`

Reply:
QQuickItem is always valid, just looses surface update so inactive?